application/emulators/Trinity_rbac_wod_soap.php

```diff
@@ -27,8 +27,7 @@ class Trinity_rbac_wod_soap implements Emulator
 	 * Array of expansion ids and their corresponding names
 	 */
 	protected $expansions = array(
-		8 => "Battle for Azeroth",
-		7 => "Legion Allied Races",
+		7 => "Battle for Azeroth",
```

Comment:
This kind of change doesn't belong in a CSS error fix PR. Make another PR for removing Legion Allied Races.